packages/server/graphql/traceGraphQL.ts

```diff
@@ -220,7 +240,8 @@ function wrappedResolve(
     }
 
     const field = assertField(tracer, context, info, path)
-
+    const tags = addVariableTags(args, info.fieldName, config.excludeArgs)
+    if (tags) field.span.addTags(tags)
```

Comment:
I'm putting them here on the resolve function vs. putting them on the top of the query.
This is because the mutation name (AKA operationName) is defined by the client (e.g. LoginMutation). If we change/add/remove that name on the client, i don't want to accidentally send sensitive info to dd.

It's also nice because on extra large queries with a TON of variables you only see the variables that make it to the specific field.